dataline-commons/src/main/java/io/dataline/commons/resources/Resourzes.java

```diff
@@ -22,32 +22,19 @@
  * SOFTWARE.
  */
 
-package io.dataline.workers.singer;
+package io.dataline.commons.resources;
 
-import io.dataline.workers.Worker;
-import java.nio.file.Path;
-import java.util.concurrent.TimeUnit;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import com.google.common.io.Resources;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.charset.Charset;
 
-public abstract class BaseSingerWorker<InputType, OutputType> implements Worker<InputType, OutputType> {
+public class Resourzes {
```

Comment:
naming ideas here? Don't want to conflict `Resources`.

Reply:
`MoreResources`